stable/chaoskube/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: chaoskube
-version: 0.6.2
-appVersion: 0.6.1
+version: 0.7.1
```

Comment:
The chart `version` is versioned somewhat independently from the `appVersion` I believe. So this should rather be:
```
version: 0.7.0
appVersion: 0.7.1
```
@unguiculus can you confirm if this is right?

Reply:
If we're bumping appVersion to 0.8.0 per the comment below, I'll bump both version strings to `0.8.0` for parity, unless there are objections